File: 240504/시험 통과 여부 확인하기/verify-test-passed.py
# if else 조건문

# if 조건에 해당하지 않는 경우 특정 코드 수행하도록

# if 조건:
#     조건이 참일 경우 수행되는 코드 작성
# else:
#     조건이 거짓일 경우에 수행되는 코드 작성
# 이 부분은 조건과 무관하게 항상 수행됨

# a = int(input())

# if a > 10:
#     a += 5
# else:
#     a *= 3
# print(a)


# 문제
x = int(input())

if x >= 80:
    print("pass")
else:
    print(f"{80 - x} more score")

File: 250117/시험 통과 여부 확인하기/verify-test-passed.cpp
// // <if else 조건문>

// // else라는 구문을 이용하면, if 조건에 해당하지 않는 경우에만 특정 코드를 수행하게 만들 수 있음
// if (조건) {
  
//   여기에 조건이 참일 경우에만 수행되는 코드 작성

// }
// else {

//     여기에 조건이 거짓일 경우에만 수행되는 코드 작성

// }

// 이 위치에 있는 코드는 조건과 무관하게 항상 수행됩니다.

// #include <iostream>
// using namespace std;

// int main() {
    
//     int a;
//     cin >> a;

//     if (a > 10) {
//         a += 5;
//     }
//     else {
//         a *= 3;
//     }

//     cout << a;
//     return 0;
// }

// <문제: 시험 통과 여부 확인하기>

#include <iostream>
using namespace std;

int main() {
    int n;
    cin >> n;

    if (n >= 80) {
        cout << "pass";
    }
    else {
        cout << 80 - n << " more score";
    }

    return 0;
}